XActivity.py

```diff
@@ -87,4 +87,26 @@ def read_file(self, file_path):
         pass  # self.game.read_file(file_path)
 
     def write_file(self, file_path):
-        pass  # self.game.write_file(file_path)
+        hscore, score, game = self.game.save_game()
+        self.metadata['hscore'] = str(hscore)
+        self.metadata['score'] = str(score)
+        # self.metadata['level'] = str(level)
```

Comment:
If you're not saving the level, how about removing the commented line of code.

Reply:
this too is probably from older commits. I've fixed it later.

Reply:
Okay, thanks!